src/main/resources/entity-types/inventory/simple_instance.json5

```diff
@@ -1012,6 +1012,10 @@
       queryable: true,
       visibleByDefault: false,
       valueGetter: ":sourceAlias.jsonb->>'staffSuppress'",
+      filterValueGetter: "COALESCE(\
+        lower(:sourceAlias.jsonb->>'staffSuppress'::text),\
```

Comment:
Nit: I'm like 97.38% sure the `::text` isn't needed

Reply:
I believe you completely and updated it.  @mweaver-ebsco